vktrace/vktrace_replay/vkreplay_vkdisplay.cpp

```diff
@@ -29,6 +29,10 @@
 #include <linux/input.h>
 #endif
 
+#if defined PLATFORM_LINUX && defined ANDROID
```

Comment:
I realize this is consistent with the lines above, but generally wrapping the values being checked in parentheses makes this much easier to read.  There are some examples below.

Reply:
Agreed about readability, but as noted the code was written this way to be consistent with the existing adjacent code.  I would like to do cleanup for style consistency in a separate PR.